### PR TITLE
Making actions menu scrollable

### PR DESCRIPTION
**Pull request by xii69**

### CODE REVIEW

**Comment by xii69** [comment]
Well the reason i did this is in some low resolution monitors you cannot see all of the list (for example my monitor which its 1600x900)
Also lowering the zoom makes everything weird.

**Comment by DevLeoko** [comment]
Hey @xii69, thanks for the PR, but changing the max-height of the floating menus for all screen sizes is probably not the right solution. On larger screens where there is enough screen space, the full height is preferable over a scrolling view.

The best solution would be to compute the available height when the menu is opened and then dynamically set the corresponding max-height. This requires some more programming knowledge and might be a bit more tedious as the floating menus are implemented in a few different places and not extracted into a single component.

A simpler solution would be to use CSS media queries to only set the max height on smaller screen sizes (e.g. < 1080px screen height)

**Comment by xii69** [comment]
OK then, i changed the code to only apply the scroll on small screens (< 1080p)

File: src/scss/app.scss [hunk 1]
```diff
@@ -117,6 +117,11 @@ a {
   transition-duration: 150ms;
   transition-property: opacity;
 
+  @media screen and (max-width: 1080px) {
+    max-height: 300px;
```
Comment (Syrent) [inline src/scss/app.scss]: I was just looking around and saw this PR, I don't specifically know what problem you trying to solve but I think a fixed height like `300px` might not be the best solution, maybe using the view height property be the best thing to do like this:
```suggestion
    max-height: 40vh;
```

File: src/scss/app.scss [hunk 2]
```diff
@@ -146,6 +151,7 @@ a {
   }
 }
 
+
```
Comment (Syrent) [inline src/scss/app.scss]: Extra line?
```suggestion

```

**Review by Syrent — CHANGES_REQUESTED** [review]
I was just looking around and saw this PR, I don't specifically know what problem you trying to solve but I think a fixed height like `300px` might not be the best solution.

File: src/scss/app.scss (hunk 1 above)
Comment (DevLeoko) [reply src/scss/app.scss]: Ya, vh is better suited, thanks

**Comment by DevLeoko** [comment]
I have changed the max-height to 50vh (50% of screen height) and used a media query based on the screen height, not width. Should be good now, thanks to you two for contributing.

**Pull request merged**